src/components/date/date.js

```diff
@@ -507,7 +507,12 @@ class Date extends React.Component {
     const datePicker = this.state.open ? this.renderDatePicker() : null;
 
     return (
-      <div className={ this.mainClasses } onClick={ this.handleWidgetClick } { ...tagComponent('date', this.props) }>
+      <div
+        className={ this.mainClasses }
+        onClick={ this.handleWidgetClick }
+        role='presentation'
```

Comment:
I'm not sure this is needed, as a `<div>` doesn't have any native semantics e.g. in the same way that a `<h1>` or `<ul>` does.

Reply:
I've removed this role